pkg/static/login.js

```diff
@@ -164,17 +164,31 @@
     }
 
     function requisites() {
-        function disableLogin(name) {
+        function disableLogin(name, bypass) {
             if (name === "supports")
                 name = "@supports API";
             const errorString = format(_("This web browser is too old to run the Web Console (missing $0)"), name);
 
             if (window.console)
                 console.warn(errorString);
             id("login-error-message").textContent = errorString;
-            hide("#login", "#login-details");
             show("#unsupported-browser", "#error-group");
-            document.body.classList.add("brand-unsupported-browser");
+            document.body.classList.add("unsupported-browser");
+
+            if (bypass) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17774-20220927-160506-c3e00978-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)